README.md

```diff
@@ -1,41 +1,40 @@
 
-## Project Description
+The Google CloudPlatform deployment for the CDOT Planned Event WZDx translator is outlined below:
+![GCP Planned Events](wzdx/docs/CDOT%20WZDx%20translators%20-%20Planned%20Events.png)
 
-This is an open source, proof of concept solution for translating work zone data in the form of CDOT Planned Events to the standardized WZDx 4.2 format, as well as having additional translators to translate COtrip/Salesforce, iCone, and NavJOY messages to the WZDx 4.2 format. This project was developed for CDOT. A unique translator has been developed for each of these message types. These translators read in the source message, parse out specific fields, and generate a WZDx message. For more information on these message formats and the data mappings between these messages and the WZDx format, see the [documentation](wzdx/docs). sample_files are located [here](wzdx/sample_files). All these translators are built to run from the command line and from GCP Dataflows, hosted within the CDOT OIM WZDX environment, connected to the RTDH (real time data hub).
+On top of these translators, combination scripts have been written to integrate WZDx messages from multiple separate sources, including CDOT Planned Events, iCone, and NavJOY. Each of these sources has unique data fields that are not present in the other sources, and the combination script is designed to merge these fields into a single WZDx message. The combination script is also built to run from the command line and from GCP Dataflows, hosted within the CDOT RTDH environment. The combination script is designed to run after the individual translators have generated their WZDx messages, and the output of the combination script is a single WZDx message that contains all of the data from the individual sources. When fields from multiple sources overlap, the combination script is designed to prioritize the data from the source with the most complete information. Geotab data is also integrated for mobile work zones, so that when a vehicle is detected in a "planned-moving-area" work zone, the work zone is changed to a "moving" work zone, using the current location of the vehicle plus a buffer distance ahead.
 
-The Google CloudPlatform deployment is outlined below.
-![GCP Processing](wzdx/docs/CDOT%20WZDx%20translators%20-%20Planned%20Events.png)
+The Google CloudPlatform deployment for the CDOT WZDx combination workflow is outlined below:
+![GCP Combination](wzdx/docs/CDOT%20WZDx%20translators%20-%20Processing.png)
 
-This project is also a full python package hosted on [pypi](https://pypi.org/project/wzdx-translator-jacob6838/)
+## Building as a Package
+
+This project is set up to be build into a python package, using python 3.8 and above. Use the following script to build the package:
 
 ```
-stuff.js
-|- public
-   |- stuff.js
+pip install wheel
```

Comment:
Is this wheel package just needed for creating the package and not by the app and that's why we don't add it to requirements.txt?

Reply:
Yes, this is only used for building the python package